Rakefile

```diff
@@ -0,0 +1,19 @@
+task :generate do
+  require 'open-uri'
+  require 'fileutils'
+  content = open("https://raw.githubusercontent.com/fastlane/fastlane/master/Rakefile").read
+  content = content.split("desc").first # only the first lines are relevant
+  eval(content)
```

Comment:
Hm, the REST API is interesting, but maybe overkill for a script like this?

It is not satisfying, but I think listing the tools again is reasonable. If we remember to come back here to use this script to generate another short link for a new tool, we can remember to update the local list of tools.

Obviously this is all just a suggestion so go with whatever feels right to you :+1: